packages/cli/src/commands/new/project.ts

```diff
@@ -49,19 +49,19 @@ export default class Project extends Command {
   public static args = [{ name: 'projectName' }]
 
   public async run(): Promise<void> {
-    return this.runWithErrors().catch(console.error)
```

Comment:
Maybe adding an `await` here could have solved the problem too, but anyway, I like your version, it's clearer!

Reply:
No, it would have output the same error, because the problem was that the ExitError was being catch by `.catch(console.error)`, while if it were outside of it, it would be catch up by the Oclif Error Handler which by default does not output ExitErrors with value 0.